### DISCUSSION TITLE
Self learning not showing completed

### DISCUSSION THREAD

**Discussion by gouravrana7** [github/india#728] [General]
I have completed all self learning but it not showing...please complete it because it gives credit in certificate...

**Comment by FRTSupport** [comment]
If your self learning course 'mandatory module' status shows "in progress", please make sure that your MS learn profile URL is entered correctly in the orientation page. Also, ensure that you are logged in to your MS learn id as well as your FRT portal while doing a course, it's important for your progress to be reflected in the Badges and Trophies section of your MS learn profile.
